### ISSUE TITLE
Please provide prebuilt binaries for v2.1.0

### ISSUE THREAD

**Issue by fgtham** [Nukesor/pueue#335]
Thanks for writing pueue! Could you please provide prebuilt binaries for v2.1.0, just like you did for the previous versions? It would make installation just so much easier :) Thanks!

**Comment by Nukesor** [comment]
Hi!

This is usually done automatically in the release build pipeline, but the artifact generation failed :/.
(I'm not sure if you can see this action): https://github.com/Nukesor/pueue/actions/runs/2711823567

There seems to be a bug in the current [UPX](https://github.com/upx/upx) version, which crashes the build process with the current binary.

```
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2020
UPX 3.96        Markus Oberhumer, Laszlo Molnar & John Reiser   Jan 23rd 2020
        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
upx: target/x86_64-unknown-linux-musl/release/pueue: CantPackException: bad DT_HASH nbucket=0  len=0x30
Packed 0 files.
(node:8226) UnhandledPromiseRejectionWarning: Error: The process '/tmp/upx-action-YUnXOe/upx-3.96-amd64_linux/upx' failed with exit code 1
Error: The process '/tmp/upx-action-YUnXOe/upx-3.96-amd64_linux/upx' failed with exit code 1
    at ExecState._setResult (/home/runner/work/_actions/svenstaro/upx-action/v2/node_modules/@actions/exec/lib/toolrunner.js:574:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/svenstaro/upx-action/v2/node_modules/@actions/exec/lib/toolrunner.js:557:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/svenstaro/upx-action/v2/node_modules/@actions/exec/lib/toolrunner.js:451:27)
    at ChildProcess.emit (events.js:314:20)
    at maybeClose (internal/child_process.js:1022:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
(node:8226) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8226) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I simply didn't have the time to debug and fix this issue yet. Also CI debugging is one of those things that are just annoying :D    

I'm sorry for the inconvenience. I might go ahead and publish uncompressed version via a new manual github action until then. The UPX release cycle is rather slow to put it mildly.

**Comment by Nukesor** [comment]
I just tried to reproduce the error, but the error doesn't show up on my local machine, even though I use the same `upx` binary, rust version and commit of pueue... I'm rather confused.

The CI has been a lot more flaky lately as well, maybe Github changed something in their worker setup?

**Comment by Nukesor** [comment]
Fixed via [61b7eb5](https://github.com/Nukesor/pueue/commit/61b7eb536b9a4a6ebca214e25159197ee20ff72e)
